src/api.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios'
+
+const newsApi = axios.create({baseURL: 'https://ncnewshostserver.onrender.com/api'})
+
+const getArticles = ()=>{
+    return newsApi
+    .get("/articles")
+    .then((response) => {
+        return response.data;
+    });
```

Comment:
You could also include a catch block here and console.log any errors that are returned from the axios request